libr/include/r_vector.h

```diff
@@ -91,16 +91,14 @@ R_API void *r_vector_shrink(RVector *vec);
 
 // RPVector
 
-// initialize vec and set vec->free to free
 R_API void r_pvector_init(RPVector *vec, RPVectorFree free);
 
-// allocate a new RPVector and set vec->free to free
 R_API RPVector *r_pvector_new(RPVectorFree free);
 
-// clear the vector and call vec->free on every element.
+// clear the vector and call vec->v.free on every element.
 R_API void r_pvector_clear(RPVector *vec);
 
-// free the vector and call vec->free on every element.
+// free the vector and call vec->v.free on every element.
 R_API void r_pvector_free(RPVector *vec);
 
 static inline size_t r_pvector_len(const RPVector *vec)					{ return vec->v.len; }
```

Comment:
Yeah it looks weird on github because tabs are 8 spaces there. If it is displayed as 4 it looks much better and the opening brackets line up vertically.

Reply:
No, it's 4: https://github.com/radare/radare2/blob/master/DEVELOPERS.md#code-style
But I don't really care. I'll change the formatting there when I have time.

Reply:
WUUT? Who changed that? :)

Reply:
https://github.com/radare/radare2/pull/9670

Reply:
Yeah I found it. I think it will go back to 8 :D but anyway, it doesn't really matter in this case. I think the main problem is that having everything on one line in such cases decreases readability.